### PR TITLE
[12.0][MIG] sale_shipping_info_helper

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: sale_shipping_info_helper/tests/test_sale_shipping_info_helper.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    def setUp(self):
+        super(TestDeliveryCost, self).setUp()
+        self.tax_model = self.env['account.tax']
+        self.SaleOrder = self.env['sale.order']
+        self.SaleOrderLine = self.env['sale.order.line']
+
+        self.partner_18 = self.env.ref('base.res_partner_18')
+        self.pricelist = self.env.ref('product.list0')
+        self.product_4 = self.env.ref('product.product_product_4')
+        self.product_uom_unit = self.env.ref('product.product_uom_unit')
```
Comment (nikul-serpentcs) [inline sale_shipping_info_helper/tests/test_sale_shipping_info_helper.py]:
```suggestion 
        self.product_uom_unit = self.env.ref('uom.product_uom_unit')
```

File: sale_shipping_info_helper/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        store=True)
+    item_amount_tax = fields.Float(
+        compute='_compute_shipping',
+        dp=dp.get_precision('Account'),
+        store=True)
+
+    @api.depends('amount_total', 'amount_untaxed')
+    def _compute_shipping(self):
+        for record in self:
+            record.shipping_amount_untaxed = 0
```
Comment (nikul-serpentcs) [inline sale_shipping_info_helper/models/sale_order.py]:
```suggestion
            record.shipping_amount_untaxed = record.shipping_amount_total = record.shipping_amount_tax = 0
```

File: sale_shipping_info_helper/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            record.shipping_amount_total = 0
+            record.shipping_amount_tax = 0
+            for line in record.order_line:
+                if line['is_delivery']:
+                    record.shipping_amount_untaxed = line['price_subtotal']
```
Comment (nikul-serpentcs) [inline sale_shipping_info_helper/models/sale_order.py]: @rousseldenis  here better to use `.get()`
record.shipping_amount_untaxed = line.get('price_subtotal', 0.0)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Review by lmignon — APPROVED** [review]
LGTM :+1:  Code + test

**Review by Laurent-Corron — APPROVED** [review]
LGTM. Works in production as expected

**Comment by rousseldenis** [comment]
@nikul-serpentcs @Nikul-Chaudhary Could you update your review ?

**Comment by sebastienbeau** [comment]
oca-bot merge minor

**Comment by rousseldenis** [comment]
@sebastienbeau When it''s a migration PR, there is no need of 'minor' as it will change version to 12.0.1.1.0 which is not necessary. Just '/ocabot merge' is sufficient

**Comment by rousseldenis** [comment]
Could you retry (ocabot without hyphen)?

**Pull request merged**